scripts/simplify_network.py

```diff
@@ -370,11 +371,11 @@ def split_links(nodes):
             n.mremove("Link", all_links)
 
             static_attrs = n.components["Link"]["attrs"].loc[lambda df: df.static]
-            for attr, default in static_attrs.default.iteritems():
+            for attr, default in static_attrs.default.items():
                 params.setdefault(attr, default)
-            n.links.loc[name] = pd.Series(params)
+            n.links.loc[name] = params
 
-    logger.debug("Collecting all components using the busmap")
+            # n.add("Link", **params)
```

Comment:
what about these loose comments?

Reply:
Restored; they have been refuses kept in the attempts to harmonize to the pypsa-eur version.
That harmonization may be done in another PR